src/hooks/useHighlight.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useRef, useEffect } from 'react'
+
+interface Result {
+  highlight: boolean
+  triggerHighlight(): void
+}
+
+export const useHighlight = (): Result => {
+  const [highlight, setHighlight] = useState(false)
+  const mounted = useRef(true)
+
+  useEffect(() => {
+    return function cleanUp(): void {
+      mounted.current = false
+    }
+  }, [])
+
+  const triggerHighlight = (): void => {
+    if (mounted.current) {
+      setHighlight(true)
+      setInterval(() => {
```

Comment:
I didn't get to check this PR in time, but...
Why `Interval`, why not `Timeout`
And if you really need Interval, need also to clean it up on unmount, as many times as you `triggerHighlight`, otherwise memory leak.

Reply:
My bad, I have been forgetting to wait for your input.

As for your suggestion, I'll create a new PR to address it.

Reply:
Yep, let's try to get the 2 approvals, leaving some fair time for review.
We can always poke the person in the daily report or chat if we are waiting for a PR and we really want to merge

Reply:
https://github.com/gnosis/dex-react/pull/112